packages/notifi-react-card/lib/hooks/useIntercomChat.ts

```diff
@@ -27,15 +27,17 @@ export type FeedEntry = {
   timestamp: string;
 } & MessagesBlockFeedEntry;
 
-type GetConversationMessagesInputProps = Readonly<{
+type UseIntercomChatProps = Readonly<{
   first?: number;
+  userId?: string;
   conversationId: string;
 }>;
 
 export const useIntercomChat = ({
   first = 5,
   conversationId,
-}: GetConversationMessagesInputProps) => {
+  userId = '0ff9528b-9bd4-444f-adcb-482673b31c96',
```

Comment:
Hmm, would it be cleaner to pass the parameters that are not used in every single function in the respective functions that are using it instead of top level? 

For example, `userId` and first are only being used by the `getConversationMessages` function. So should we only ask the parameter values at the function level instead at the hook level?

`const getConversationMessages = useCallback((first, userId))`

Reply:
Yeah, I agreed we could pass the param directly in the function. but as the params we have(conversationId, first, userId) will always be the same value during the entire flow, it will be more convenient for us to pass the params in when using the useIntercomChat hook, and we don't need to pass the value every time when calling the function. For example, if we need to call getConversationMessages twice from the useIntercomChat, and we didn't have the params passed in the top level, we have to pass the param every time that we call getConversationMessages.

Reply:
Hmm, I think conversationId and userId is probably okay since it is intuitive to a dev that it is on the top level since it pertains to the conversation as a whole and as you mentioned, might be variables that are used quite regularly.

Maybe we can still pass `first` an optional parameter at the function level (that defaults to 5 if nothing is passed in) because it is very specific to `getConversationMessages` and I don't see it as a parameter that will need changed.

Actually, what is even better is to have a default number at the query level. That way, you can make parameter optional and not have to pass in a default value at the component level because you already assign a default at the query level. 

Something like:

```
const QUERY = `query conversationMessages(
  $getConversationMessagesInput: GetConversationMessagesInput!
  $after: String
  $first: Int = 5
```

Reply:
updated to pass first as an optional parameter at the function level(getConversationMessage).